### DISCUSSION TITLE
Trying to override selection color

### DISCUSSION THREAD

**Discussion by ldijkman** [ajaxorg/ace#5308] [Q&A]
Hello

in some themes selection is hard to see
mouse selection or search selection

Trying to override selection color

gutter and background i can override color now
but no luck yet with selection color

```
<script>
  function setcolor(setcolor){
    console.log("setcolor",setcolor);
    editor.container.style.background=setcolor; 
  }

  function setcolorgutterbg(setcolor){
    console.log("setcolor",setcolor); 
    document.querySelector(".ace_gutter").style.background=setcolor;
  }
// selection color change is not working ok
  function setcolorselection(setcolor){
    console.log("setcolor",setcolor); 
    document.querySelector(".ace_marker-layer .ace_selection").style.background=setcolor;
    document.querySelector(".ace_marker-layer .ace_selected-word").style.background=setcolor;
     
  }
</script>
```

hope someone can help me?!

https://plnkr.co/edit/Digj5Fw7D6hxaTU4
![2023-09-10-063337_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/27ce355a-da8c-43ba-84d5-becd8ff23e19)

---
https://plnkr.co/edit/Digj5Fw7D6hxaTU4
---
![2023-09-10-063701_1920x1080_scrot](https://github.com/ajaxorg/ace/assets/45427770/16899d97-1d66-4d0f-a82c-61416e42e261)
---
https://plnkr.co/edit/Digj5Fw7D6hxaTU4

**Comment by nightwing** [comment]
There are several dom nodes corresponding to selection, and they change when selection changes, so setting style attribute is not possible,  use css rule instead.

**Reply by ldijkman** [reply]
Thank you for your answer
Had no idea what you meant
So searched a bit on css rules
It works now
https://youtu.be/LKHT42LGuDw?si=zZCGCiub9o_UGRaZ
